tests/test_models.py

```diff
@@ -1025,11 +1031,13 @@ def test_channel_code(self):
     @mock.patch('integrated_channels.sap_success_factors.transmitters.SAPSuccessFactorsAPIClient')
     @mock.patch(
         'integrated_channels.sap_success_factors.transmitters.learner_data.SuccessFactorsLearnerDataTransmitter')
+    @mock.patch('integrated_channels.integrated_channel.learner_data.EnrollmentApiClient')
     @mock.patch('integrated_channels.integrated_channel.learner_data.CertificatesApiClient')
     @mock.patch('integrated_channels.integrated_channel.learner_data.CourseApiClient')
     @mock.patch('enterprise.lms_api.JwtBuilder', mock.Mock())
-    def test_transmit_learner_data(self, mock_course_api, mock_certificate, mock_transmitter, mock_sap_client):
-
+    def test_transmit_learner_data(
+            self, mock_course_api, mock_certificate, mock_enrollment, mock_transmitter, mock_sap_client
```

Comment:
@UmanShahzad Not directly related to the task at hand, but since you're touching this: I think it would be worth making parameter names consistent, i.e., append `_api` to any parameter that represents a patched `ApiClient`.